Added star/unstar option in messageActionSheet.

vishwesh3 opened the pull request:
fix:#706

borisyankov commented on src/api/starMessage.js:
```diff
@@ -0,0 +1,7 @@
+/* @flow */
```
Not a big fan of the naming conflict with the event handlers.
Why not call the API call 'toggleMessageStarred' and set the third parameter being boolean - true = star, false = unstar?

vishwesh3 replied:
Done :+1:

borisyankov commented:
Looks good, I had just one comment.

vishwesh3 commented:
Updated branch.
Added another commit: https://github.com/zulip/zulip-mobile/pull/715/commits/e2e4d2c7725ac505e22ebbd3997eda613bf82a3e which resolve app crash when this objects (flags) are accessed (mainly on freshly installed app).

borisyankov commented:
Looks good. Thanks.
